ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -300,6 +300,25 @@ echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 openshift-install --dir="${dir}" create manifests &
 wait "$!"
 
+# Adds arm64 workers for heterogeneous deployment
+if [[ "${HETEROGENEOUS:-}"  == 'true' ]]; then
+  if [ ! -f /tmp/yq ]; then
```

Comment:
should we add a condition like

```
if [ ! -f /tmp/yq ] && [ $(/tmp/yq -V | sed 's/^.*version \([0-9]*\)\..*$/\1/') -ge 4 ]; then
```

to be sure that we are using the proper yq version and there is no need to download it?

Reply:
yq needs to be added as part of the image. Its been utilized for other scenarios, there are different version being used by different teams and we want to confirm before changing and not breaking stuff